snap.manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
The version we have today in NPM is 1.0.0, we should not change that unless we're doing a new release

Reply:
[e932ac9](https://github.com/MetaMask/snap-watch-only/pull/12/commits/e932ac956f923ea99b1ae9b18df00b7915c80b20)